webapp/channels/src/components/admin_console/workspace-optimization/dashboard_checks/config.ts

```diff
@@ -76,6 +76,40 @@ const sessionLength = (
     };
 };
 
+/**
+ *
+ * @description This checks to see if Mattermost is running as root.
+ */
+const rootUserCheck = async (
+    config: Partial<AdminConfig>,
+    formatMessage: ReturnType<typeof useIntl>['formatMessage'],
+    options: Options,
+) => {
+    const fetchRootStatus = async () => {
+        const result = await Client4.ping(false);
```

Comment:
I would love to move the `/ping` request out of the individual checks, but that doesn't have to block this PR.

Reply:
Yes, makes sense.